tests/e2e/configurer/chain/commands.go

```diff
@@ -86,7 +86,7 @@ func (n *NodeConfig) CreateConcentratedPool(from, denom1, denom2 string, tickSpa
 func (n *NodeConfig) CreateConcentratedPosition(from, lowerTick, upperTick string, token0, token1 string, token0MinAmt, token1MinAmt int64, poolId uint64) uint64 {
 	n.LogActionF("creating concentrated position")
 
-	cmd := []string{"osmosisd", "tx", "concentratedliquidity", "create-position", lowerTick, upperTick, token0, token1, fmt.Sprintf("%d", token0MinAmt), fmt.Sprintf("%d", token1MinAmt), fmt.Sprintf("--from=%s", from), fmt.Sprintf("--pool-id=%d", poolId), "-o json"}
+	cmd := []string{"osmosisd", "tx", "concentratedliquidity", "create-position", lowerTick, upperTick, token0, token1, fmt.Sprintf("%d", token0MinAmt), fmt.Sprintf("%d", token1MinAmt), fmt.Sprintf("--from=%s", from), fmt.Sprintf("--pool-id=%d", poolId), "--gas=600000", "--fees=5000uosmo", "-o json"}
```

Comment:
Why are these values chosen specifically?

Is it possible to break them down to make it explicit where `600000` gas and `5000uosmo` are coming from, please?

Reply:
If I'm not mistaken, this should only require the .0025uosmo min fee since we are using less than 1mil gas

Reply:
these are the values closest to the ones i found on local osmosis setup.sh

Reply:
@czarcas7ic so i looked it up and the fees i set to `Fees = txfeestypes.ConsensusMinFee.Mul(sdk.NewDec(GasLimit)).Ceil().TruncateInt64()` and in this case wouldnt' it be 60000 * 0.0025 = 1500uosmo

Reply:
okay so redid this a bit. 
New param; 
gas = 50,000 (e2e was default to 40,000, i added extra 10k because that's the hard coded gas to consume per tick initialization) 
fees = 1250 (because 50,000 * 0.0025 = 1250) 

added in comments too

Reply:
Nice, this logic follows to me